frontend/src/components/trivia/trivia.tsx

```diff
@@ -1,23 +1,54 @@
 import { Button } from "@/components/ui/button";
 import { Label } from "@/components/ui/label";
 import { RadioGroup, RadioGroupItem } from "@/components/ui/radio-group";
+import { useState } from "react";
+import { useSearchParams } from "react-router-dom";
+import { useToast } from "@/components/ui/use-toast"
+
 
 type Answer = {
   id: number;
   value: string;
+}
+type quizDataProps = {
+  id: number;
+  question: string;
+  answers: Answer[];
 };
 
 type TriviaProps = {
-  question: string;
-  answers: Answer[];
-  submitHandler: () => void;
+  quizData: quizDataProps[];
 };
-const Trivia = ({ question, answers,submitHandler }: TriviaProps) => {
+const Trivia = ({ quizData }: TriviaProps) => {
+  const { toast } = useToast()
+  const [isButtonEnable, setIsButtonEnable] = useState(true);
```

Comment:
**suggestion(non-blocking) :**  can we change the name of `isButtonEnable` to `isButtonDisabled` and the setter function accordingly, I think that will more specific.

Reply:
Changes has been made as per your suggestion.